snooty.toml

```diff
@@ -52,40 +52,40 @@ atlas = "Atlas"
 
 [constants]
 admin-api-page = "/admin/api/v3/#"
-app = "App"
-apps = "Apps"
-atlas = "MongoDB Atlas"
-atlas-short = "Atlas"
+#app = "App"
+#apps = "Apps"
+#atlas = "MongoDB Atlas"
+#atlas-short = "Atlas"
 adf = "Atlas Data Federation"
```

Comment:
can you delete these instead of commenting out?

Reply:
yep! deleted.